cortex/built_ins/datasets/torchvision_datasets.py

```diff
@@ -33,23 +33,45 @@ def _handle_SVHN(self, Dataset, data_path, transform=None, **kwargs):
             data_path, split='test', transform=transform, download=True)
         return train_set, test_set
 
-    def _handle_STL(self,
-                    Dataset,
-                    data_path,
-                    transform=None,
-                    labeled_only=False):
+    def _handle_STL(self, Dataset, data_path, transform=None,
+                    labeled_only=False, stl_center_crop=False,
+                    stl_resize_only=False, stl_no_resize=False):
         normalize = transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
-        train_transform = transforms.Compose([
-            transforms.RandomResizedCrop(64),
-            transforms.RandomHorizontalFlip(),
-            transforms.ToTensor(),
-            normalize,
-        ])
-        test_transform = transforms.Compose([
-            transforms.Resize(64),
-            transforms.ToTensor(),
-            normalize,
-        ])
+
+        if stl_center_crop:
+            tr_trans = transforms.CenterCrop(64)
+            te_trans = transforms.CenterCrop(64)
+        elif stl_resize_only:
+            tr_trans = transforms.Resize(64)
+            te_trans = transforms.Resize(64)
+        elif stl_no_resize:
+            pass
+        else:
+            tr_trans = transforms.RandomResizedCrop(64)
+            te_trans = transforms.Resize(64)
+
+        if stl_no_resize:
+            train_transform = transforms.Compose([
+                transforms.RandomHorizontalFlip(),
+                transforms.ToTensor(),
+                normalize,
+            ])
+            test_transform = transforms.Compose([
+                transforms.ToTensor(),
+                normalize,
+            ])
+        else:
```

Comment:
You could bring the uppre `if`, `else` block down here.

Reply:
done